kinematics_base/src/kinematics_base.cpp

```diff
@@ -122,3 +122,35 @@ const bool kinematics::KinematicsBase::supportsGroup(const moveit::core::JointMo
 
   return true;
 }
+
+bool kinematics::KinematicsBase::getMultipleIK(const geometry_msgs::Pose &ik_pose,
+                           std::vector< std::vector<double> >& solutions,
+                           kinematics::KinematicsResult& result,
+                           const kinematics::KinematicsQueryOptions &options) const
+{
+  std::vector<double> solution;
+  std::vector<double> seed(getJointNames().size(),0.0f);
+  result.solution_percentage = 0.0f;
+
+  if(options.solutions_search_code != KinematicSearches::ONE)
+  {
+    result.kinematic_error = kinematics::KinematicErrors::UNSUPORTED_SEARCH_REQUESTED;
+    return false;
```

Comment:
Should we log something here?  Not sure if logging from a plugin is good practice.